bundle.go

```diff
@@ -122,9 +123,9 @@ func (r *DownloadRequest) downloadBundle() {
 
 func (r *DownloadRequest) checkTimeout() {
 
-	if !r.timeoutAfter.IsZero() {
-		if time.Now().After(r.timeoutAfter) {
-			r.timeoutAfter = time.Time{}
+	if !r.markFailedAt.IsZero() {
```

Comment:
In what case is r.timeoutAfter the Zero instant.  It looks like this variable is being used to indicate that a bundle failed to download